e-commerce/README.md

```diff
@@ -137,6 +137,25 @@ To access the dashboard:
 
 The dashboard provides logs and traces for all of your applications.  For additional information about the dashboard, see the [Monitoring Dashboard tutorial](https://docs.dbos.dev/cloud-tutorials/monitoring-dashboard).
 
+## Using the DBOS Debugger
+
+Provenance data is captured by DBOS during workflow execution.  The DBOS debugger uses this provenance data to provide replays of past transactions.  This novel feature allows production scenarios to be reproduced in the development environment.
```

Comment:
May need to clarify that we currently only replays cloud workflows. So we can say something like `Provenance data is automatically captured by DBOS Cloud during workflow execution -- time travel currently only works for applications deployed to DBOS Cloud.`

Reply:
I can say something like that.